### PR TITLE
Add a dump when loading the catalog in the legacy behats

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**
When we load the fixtures in the behat, we are using basic import. When we load several times the same catalog, it's pretty useless as we could use a SQL dump instead.

Actually this is what is done for the integration tests. We didn't want to complexify the stack one year ago with a cache layer with the new CI. Now that it's pretty "stable" (the infra, not the tests :trollface: ) and mature, I think it's time to use a cache.

We already did an experiment one year ago with Pierre (https://github.com/akeneo/pim-community-dev/pull/9624), but it was a lot of copy/past. A better solution is to use the same way to load a catalog between behat and integration tests.

I had to adapt tests on reference data, as it's now loaded by default with all the reference data (as in the integration tests). Before, it was not loaded.

Note: I don't think it will lead to more unstability in our tests as our issues occur **after** loading the catalog (race condition on the front 99% of the time).

**CE Results**
```
curl "https://circleci.com/api/v1.1/project/github/akeneo/pim-community-dev/27765/tests?circle-token=token_circle_ci" | jq '.tests |= sort_by(.run_time) | .tests[].run_time' | awk '{sum+=$1} END{printf("%f\n",sum)}'
```

Total: 13460 seconds (3h40)
After: 11375 seconds  (3h09)

Gain -15% time

**EE Results**

Before: 35633 seconds (9h53)
After: 26468 seconds (7h21)
Gain: -25% time

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature [hunk 1]
```diff
@@ -33,22 +30,22 @@ Feature: Execute a job
     And the family of the product "SKU-001" should be "new_heels"
     And product "SKU-002" should be enabled
     And the product "SKU-001" should have the following values:
-      | heel_color               | [Red]          |
-      | sole_fabric              | [Nylon], [PVC] |
-      | lace_fabric-en_US-tablet |                |
-      | lace_fabric-en_US-mobile | [Kevlar]       |
+      | heel_color               | [red]              |
```
Comment (jjanvier) [inline tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature]: are all those kind of things normal?
we should we change the tests if we just change the way to load the fixtures?

**Comment by jjanvier** [comment]
(Nice PRs by the way)

**Comment by ahocquard** [comment]
@jjanvier I relaunched it something like 3 times and in EE same. I think it's stable because our issues are about the race condition in the UI.

File: tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature (hunk 1 above)
Comment (ahocquard) [reply tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature]: did you read the descrioption :wink: ? Reference data are loaded with the ctaalog by default now, as for the integration tests. It was not loaded before, so I had issues about duplicated keys. It does not change the behavior of the test.

Comment (jjanvier) [reply tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature]: my comment was more about: before we had a label and now we have a code
this kind of refactoring should not change those tests (except the ones that loads data that are now here with the dump of course)

Comment (ahocquard) [reply tests/legacy/features/pim/enrichment/reference-data/import/import_products.feature]: It was a code in both cases, but before it was a code with an uppercase ;)

**Pull request merged**